problem_builder/answer.py

```diff
@@ -86,6 +90,14 @@ def get_model_object(self, name=None):
 
         return answer_data
 
+    def get_answer_data(self):
+        answer_data = self.get_model_object()
+        return {
+            "student_input": answer_data.student_input,
```

Comment:
This isn't always right - we should return `self.student_input` instead which uses https://github.com/open-craft/problem-builder/blob/de315bb040008554be3e509ae0fc82b094c15ff0/problem_builder/answer.py#L171-L188 , the correct `student_input` algorithm.